### ISSUE TITLE
Cannot edit breakpoints with screen reader enabled

### ISSUE THREAD

**Issue by francipvb** [Microsoft/vscode#66351]
Issue Type: <b>Bug</b>

This problem occurs when using Python debugger and I don't know if it occurs with other debuggers.

My primary screen reader is NVDA.

1. Add a breakpoint with F9 key.
2. In the breakpoints list, select the new added breakpoint.
3. Press SHIFT+F10 to open the context menu and select _Edit Breakpoint_.

In my system I was unable to locate any group of controls to edit the breakpoint, but I am not sure if it is displayed or it is a very visual-intuitive breakpoint, the fact is that I cann't add other types of breakpoints tha the toggled one.

VS Code version: Code 1.30.2 (61122f88f0bf01e2ac16bdb9e1bc4571755f5bd8, 2019-01-07T22:54:13.295Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz (8 x 2592)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.89GB (8.84GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|0%|

</details><details><summary>Extensions (25)</summary>

Extension|Author (truncated)|Version
---|---|---
html-snippets|abu|0.2.1
vscode-angular2-files|ale|1.6.2
vscode-django|bat|0.17.0
vagrant|bbe|0.5.0
django-snippets|bib|1.1.1
aspnetcore-angular|buz|0.0.3
path-intellisense|chr|1.4.2
bracket-pair-colorizer|Coe|1.0.61
php-debug|fel|1.12.6
php-intellisense|fel|2.3.10
php-pack|fel|1.0.2
vscode-pull-request-github|Git|0.3.2
vscode-firefox-debug|hbe|1.7.4
ftp-simple|hum|0.6.8
csharpextensions|jch|1.3.0
vscode-github|Kni|0.30.1
vscode-language-pack-es|MS-|1.30.2
python|ms-|2018.12.1
cpptools|ms-|0.20.1
debugger-for-chrome|msj|4.11.1
debugger-for-edge|msj|1.0.6
material-icon-theme|PKi|3.6.2
psi-header|psi|1.6.6
vscode-yaml|red|0.2.1
vscode-icons|rob|8.0.0

</details>

**Comment by francipvb** [comment]
Responding for completeness...

> (Experimental duplicate detection)
> Thanks for submitting this issue. Please also check if it is already covered by an existing one, like:
> 
>     * [Allow for a Breakpoint to be Enabled when breakpoints are deactivated (#65394)](https://www.github.com/Microsoft/vscode/issues/65394)
> 
>     * [Expose the Id in the base Breakpoint class (#48722)](https://www.github.com/Microsoft/vscode/issues/48722)
> 
>     * [Conditional breakpoints: up/down to navigate through condition history (#50412)](https://www.github.com/Microsoft/vscode/issues/50412)
> 
>     * [Breakpoints not visible in gutter and editor (#64117)](https://www.github.com/Microsoft/vscode/issues/64117)
> 
>     * [Show visual hints for inline breakpoints (#31612)](https://www.github.com/Microsoft/vscode/issues/31612)

I am not sure if any of these are related directly or indirectly to this.

**Comment by isidorn** [comment]
@francipvb hi. I just tried this and it works for me. So to be clear here are my steps:
1. Keyboard focus on breakpoint, press Shift + F10, get context menu choose "Edit breakpoint"
2. For me focus nicely switches to the breakpoint input widget and I can edit it (add condition, hit count)... Note that this experience is not designed to change the line number of a breakpoint, for that you should remove, add the breakpoint again.

What is different for you? Or what would you expect to be different?

**Comment by francipvb** [comment]
Any update on this?

Cheers,

**Comment by isidorn** [comment]
Sorry this somehow slipped through my rather.
@francipvb do you still see this? If yes I will investigate

**Comment by francipvb** [comment]
Hello @isidorn 

Yes, it occurs yet, checked against the latest update and some minutes ago.

Cheers,

**Comment by isidorn** [comment]
@francipvb this is happening due to a timouet which we had to introduce as a workaoround for an electron issue where focus was not properly propageted when using native context menus.
What you are describing is only reproducing on windows and not 100%.
Due to that I have simply increased the timeout from 100 to 150ms.
Please try vscode insiders next week and let us know if the issue is better.
Thank you very much!
